README.md

```diff
@@ -5,11 +5,73 @@
 Grid is a peer-to-peer network of data owners and data scientists who can collectively train AI models using [PySyft](https://github.com/OpenMined/PySyft/).
 
 
+## Overview
+- [How to install](#how-to-install)
+- [Getting Started](#getting-started)
+- [Try out the Tutorials](#try-out-the-tutorials)
+- [Start Contributing](#start-contributing)
+- [High Level Architecture](#high-level-architecture)
+- [Disclaimer](#disclaimer)
+
+## How to install
+
+#### Grid library
+```
+$ python setup.py install
+```
+#### Grid Node
+```
+$ cd app/websocket
+$ pip install -r requirements.txt
+```
+#### Grid Gateway
+```
+$ cd gateway
+$ pip install -r requirements.txt
+```
+
+## Getting started
+To boot the entire grid platform locally, we will use docker containers.  
+To install the dependencies, just follow these steps:
+
+#### Docker dependencies
```

Comment:
I think it's better to reference the installation instructions from the official Docker website as every OS has its own way of installing Docker.

Reply:
Done! (badbe85)